nextflow_schema.json

```diff
@@ -1176,13 +1176,34 @@
                     "fa_icon": "fas fa-search",
                     "enum": ["relaxed", "strict", "loose"]
                 },
+                "bgc_antismash_pfam2go": {
+                    "type": "boolean",
+                    "default": "false",
+                    "description": "Run Pfam to Gene Ontology mapping module.",
+                    "help_text": "For more information see the antiSMASH [documentation](https://docs.antismash.secondarymetabolites.org/).\n\n> Modifies tool parameter(s):\n> - antiSMASH: `--pfam2go`",
```

Comment:
No further description what it does exactly? Why would you want to run ti?

Reply:
I have no clue why I want to run it :sweat_smile: There is no documentation other than this line in `antismash --help`. But I'll add the info that @Darcy220606 provided in the respective issue.